driver-core/src/test/functional/com/mongodb/operation/DropIndexOperationSpecification.groovy

```diff
@@ -95,59 +81,49 @@ class DropIndexOperationSpecification extends OperationFunctionalSpecification {
         collectionHelper.createIndex(keys)
 
         when:
-        new DropIndexOperation(getNamespace(), keys).execute(getBinding())
+        execute(new DropIndexOperation(getNamespace(), keys), async)
         List<Document> indexes = getIndexes()
 
         then:
         indexes.size() == 1
         indexes[0].name == '_id_'
+
+        where:
+        async << [true, false]
     }
 
-    @Category(Async)
-    def 'should drop existing index asynchronously'() {
+    def 'should support maxTime'() {
```

Comment:
Can this use ClusterFixture#enableMaxTimeFailPoint?

Reply:
Done